### PR TITLE
[tests-only]Skipped 3 failed tests

### PR DESCRIPTION
**Pull request by Talank**
3 tests are failing a lot in drone, which has been skipped in this PR. 
Ref: https://github.com/owncloud/client/issues/9281#issuecomment-993317582

### CODE REVIEW

File: test/gui/tst_syncing/test.feature [hunk 1]
```diff
@@ -7,7 +7,7 @@ Feature: Syncing files
     Background:
         Given user "Alice" has been created on the server with default attributes and without skeleton files
 
-    @smokeTest
+    @smokeTest @skip
```
Comment (saw-jan) [inline test/gui/tst_syncing/test.feature:10]: Could you also mention issue number to track
```suggestion
    @smokeTest @skip @issue-0000
```

**Review by swoichha — COMMENTED** [review]
I think you need make changes in drone.star file as well

File: test/gui/tst_syncing/test.feature (hunk 1 above)
Comment (Talank) [reply test/gui/tst_syncing/test.feature]: done

Comment (Talank) [reply test/gui/tst_syncing/test.feature]: done

**Review by swoichha — APPROVED** [review]
LGTM

**Pull request merged**